app/views/cases/_event.html.erb

```
@@ -2,7 +2,7 @@
   <div class="card event-card">
     <div class="card-header d-inline-flex justify-content-between align-items-center">
       <span><%= date.to_formatted_s(:long) %> - <b><%= name %></b></span>
-      <div title="<%= details %>">
+      <div title="<%= details ||= nil %>">
```

In general, we've had a policy of "fail fast" in these situations, so that when we add new event cards we start failing tests straight away when there's information missing (`details` in this case).

That this hasn't happened implies that there's no test that tries to create an event card for change requests - so please could you add such a test and revert this line?

---

That wouldn't stop future event cards from causing issues. The point here is that when a new event card has been made without `details` it would break and unless people know to make a test for it then it could slip through like this did.